src/content/ui-api/management-api/authorization/index.md

```diff
@@ -0,0 +1,246 @@
+
+### Configure access for read-only users {#configure-read-only}
+
+A read-only user in the sense of the Management API is one that can discover or, in the Giant Swarm web UI, browse resources in order to find out about existing clusters and their configuration, as well as apps installed and available.
+
+To enable a group or user for read access to resources of one organization, you only have to bind them to these two cluster roles within the namespace of the organization:
+
+- `read-all`
+- `read-in-cluster-ns`
+
+More details on these roles are given in the [pre-defined roles](#pre-defined-roles) section.
+
+The manifest below shows how the according role bindings could be created.
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: bind-read-all-to-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: read-all
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: bind-read-in-cluster-ns-to-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: read-in-cluster-ns
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+```
+
+Our [web UI]({{< relref "/ui-api/web/organizations/access-control" >}}) makes it easy to create these role bindings interactively. Go to **Organizations**, select the organization, and navigate to the **Access control** tab. Then, one by one, select the pre-defined roles in the left column and add the user or group in the **Subjects** tab on the right.
+
+### Configure access for app developers {#configure-app-developer}
+
+Compared to a read-only user, this type of user would gain permission to install apps in workload clusters, change any app's configuration, and remove the app. To achieve this, instead of the `read-in-cluster-ns` role we bind `write-in-cluster-ns`.
+
+The according example manifest:
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: bind-read-all-to-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: read-all
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: bind-write-in-cluster-ns-to-group
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: write-in-cluster-ns
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+```
+
+Like in the case of the read-only user, these bindings can be set up easily via the web UI.
+
+### Configure access for organization admins {#configure-org-admins}
+
```

Comment:
For organization admins, we've only included instructions to bind the `cluster-admin` role. Do we still need to bind the `write-in-cluster-ns` role separately to allow them access to app resources in the workload clusters?

Reply:
Yes! This is done.